src/sentry/audit_log/register.py

```diff
@@ -70,7 +70,7 @@
         event_id=32,
         name="PROJECT_REMOVE",
         api_name="project.remove",
-        template="removed project {slug}",
+        template="removed project {slug} in {origin}",
```

Comment:
```suggestion
        template="removed project {slug}" + ("" if origin == "unknown"  else " in {origin}")
```

Reply:
@priscilawebdev this is a possible solution

Reply:
oh that simple 😄 thanks @iambriccardo